src/Sql/Sql/Database Backup/Cmdlet/RestoreAzureRMSqlDatabase.cs

```diff
@@ -295,6 +294,14 @@ public class RestoreAzureRmSqlDatabase
             HelpMessage = "The zone redundancy to associate with the Azure Sql Database. This property is only settable for Hyperscale edition databases.")]
         public SwitchParameter ZoneRedundant { get; set; }
 
+        /// <summary>
+        /// Gets or sets the tags associated with the Azure Sql Database
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The tags to associate with the Azure Sql Database")]
+        [Alias("Tag")]
+        public Hashtable Tags { get; set; }
```

Comment:
Please use Tag to avoid signature issue.

 "AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
  "Az.Sql","Microsoft.Azure.Commands.Sql.Backup.Cmdlet.RestoreAzureRmSqlDatabase","Restore-AzSqlDatabase","1","8410","Parameter Tags of cmdlet Restore-AzSqlDatabase does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."